12LessonTwelve/homework/index.html

```diff
@@ -1,10 +1,66 @@
 <!DOCTYPE html>
 <html>
+
 <head>
-  <title></title>
-  <link rel="stylesheet" href="css/style.css">
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Bootstrap Lesson12 Homework</title>
+    <!-- Google font-->
+    <link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" rel="stylesheet">
+    <link rel="stylesheet" href="css/style.css">
 </head>
+
 <body>
+    <section class="container">
+        <div class="row">
+            <div class="col-sm-12 text-center">
+                <div class="header">
+                    <h1>Best Selling Products</h1>
+                </div>
+            </div>
+        </div>
+    </section>
 
+    <main>
+        <section id="sales">
+            <div class="container-fluid">
+                <div class="row">
+                    <div class="box1 col-lg-3 col-sm-6 col-xs-12 text-center">
+                        <div class="happy">
+                            <span class="price">$35.00</span></div>
+                        <div>
+                            <h2>Happy Ninja</h2></div>
+                        <div class="cart"><a href="#">Add to Cart</a></div>
+                    </div>
+                    <div class="box2 col-lg-3 col-sm-6 col-xs-12 text-center">
+                        <div class="patient">
+                            <span class="price">$35.00</span></div>
+                        <div>
+                            <h2>Patient Ninja</h2></div>
+                        <div class="cart"><a href="#">Add to Cart</a></div>
+                    </div>
+                    <div class="box3 col-lg-3 col-sm-6 col-xs-12 text-center">
+                        <div class="ideas">
+                            <span class="price">$35.00</span></div>
+                        <div>
+                            <h2>Ship Your Idea</h2></div>
+                        <div class="cart"><a href="#">Add to Cart</a></div>
+                    </div>
+                    <div class="box4 col-lg-3 col-sm-6 col-xs-12 text-center">
+                        <div class="flying">
+                            <span class="price">$35.00</span></div>
+                        <div>
+                            <h2>Flying Ninja</h2></div>
+                        <div class="cart"><a href="#">Add to Cart</a></div>
+                    </div>
+                </div>
+            </div>
+        </section>
+    </main>
+
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js" /></script>
 </body>
+
```

Comment:
Hi Luke, I don't know why this looks ok on my screen!





--------------------------------
On Tue, 9/13/16, Ethan Stenis <notifications@github.com> wrote:

 Subject: Re: [AustinCodingAcademy/intro-workbook] Austin.intro.2.Dukette.Barbara (#92)
 To: "AustinCodingAcademy/intro-workbook" <intro-workbook@noreply.github.com>
 Cc: "Bjdukette" <Bjdukette@yahoo.com>, "Author" <author@noreply.github.com>
 Date: Tuesday, September 13, 2016, 6:07 PM
 
 In 12LessonTwelve/homework/index.html:
 >  </body>
 > +
 
 Looks like that pricing button is going crazy! Let's
 get that settled into place and I can bump your grade up.
 The Dev Tools is a great way to troubleshoot this issue.
 
 
 
 —
 You are receiving this because you authored the thread.
 Reply to this email directly, view
 it on GitHub, or mute
 the thread.